### ISSUE TITLE
[TalkBack] does not work properly in new Google Play Music app

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#265]
```
What steps will reproduce the problem?
1. Make sure you're using PlayMusic 5.0 released a few days ago
2. Open Play Music
3. nearly every UI interface elements are spoken as "Need to do accessibility"

What is the expected output? What do you see instead?

I think that Play Music team used a new type of widget that need to be made 
accessible for talkback to read properly artist/album/track name/whatever, and 
interact with them as appropriate.


What version of the product are you using? On what operating system?
Android 4.2, nexus 4, talkback 3.3 (last playstore release), google play music 
5.0.

Please provide any additional information below.

This issue is also relevant when using the new Play Books app
```

Original issue reported on code.google.com by `plassiar...@gmail.com` on 17 May 2013 at 12:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 5 Dec 2013 at 9:56
- Changed state: **Fixed**
